### ISSUE TITLE
train_README-ja 誤表記/ train_README-ja Typo

### ISSUE THREAD

**Issue by FA7770** [kohya-ss/sd-scripts#1446]
train_README-jaドキュメント内の任意のスケジューラを指定するオプション引数表記に誤りがあるようです。  
正: `--lr_scheduler_args`  
誤: `--scheduler_args`

It seems there is a typo in the option argument notation for specifying an arbitrary scheduler in the train_README-ja document.  
Correct: `--lr_scheduler_args`  
Incorrect: `--scheduler_args`

https://github.com/kohya-ss/sd-scripts/blob/25f961bc779bc79aef440813e3e8e92244ac5739/docs/train_README-ja.md?plain=1#L651
